config.json

```diff
@@ -342,6 +354,24 @@
       "values": [],
       "defaultValue": ""
     },
+    "OptionalToolSource": {
```

Comment:
Do you actually need these here? Do they work with if you pass them after "--"? I ask because I don't see them being passed through the scripts.